.github/workflows/gradle-build-development.yml

```diff
@@ -1,7 +1,9 @@
 name: Gradle Build & Deploy - Develop
 on:
   push:
-    branches: [develop]
+    branches:
+      - 'develop'
+      - 'feature-1112/*'
```

Comment:
is this intentional ? don't we ned this script for all feature-* branches or this is only to build and test 1112 ?

Reply:
This only triggers for branches related to issue #1112. (basically this branch, I was just lazy.) I'll be removing that trigger before merge anyway.